manifests/install.pp

```diff
@@ -49,19 +49,19 @@
     }
   }
 
-  if $::osfamily == 'windows' {
+  if $facts['osfamily'] == 'windows' {
     # required to install telegraf on windows
     require chocolatey
 
     # package install
-    package { $::telegraf::package_name:
-      ensure          => $::telegraf::ensure,
+    package { $telegraf::package_name:
+      ensure          => $telegraf::ensure,
       provider        => chocolatey,
```

Comment:
Should this be quoted?

Reply:
Style-wise? Not sure. It is valid though as is. https://puppet.com/docs/puppet/5.2/lang_data_string.html#bare-words